### ISSUE TITLE
Handle page numbering in the merge PDFs route

### ISSUE THREAD

**Issue by KojoEnch** [gotenberg/gotenberg#964]
When merging several PDFs through the [Merge PDFS route](https://gotenberg.dev/docs/routes#merge-pdfs-route), it would be really useful to have **the ability to reset the numbering of pages in the merged file**.

For instance, in order to generate a PDF with a different header for each chapter, we first generate partial PDFs and then merge them. The issue is that page numbering in the merged footer keeps the inherited numbering, i.e. restarting at 1 for each chapter.

Would it be feasible to address this issue ? I'm thinking of an _optional footer_ or a _resetNumbering form field_ accepted by the merge route. 

Thank you for the awesome work.

**Comment by gulien** [comment]
Hello @KojoEnch 👋 

That would make a lot of sense, but I’m not sure this feature is available in the underlying tools used by Gotenberg. For instance, I did not find anything with QPDF.

**Comment by KojoEnch** [comment]
@gulien thanks for your  reply.

I installed QPDF, read the docs and tried a few CLI. At one point, I thought I had found the Graal with the ` --set-page-labels` option, then I learned PDF page labels are NOT page numbers in footer ! :expressionless:  

If you suggest any other library used in this repo where we might find something useful, I'd be glad to continue searching.

If renumbering while merging ends difficult to implement, maybe a kinda _replace footer_ route after merging ?
